parsing/lexer.mll

```diff
@@ -377,7 +378,11 @@ rule token = parse
         COMMENT (s, loc) }
   | "(**"
       { let s, loc = with_comment_buffer comment lexbuf in
-        DOCSTRING (Docstrings.docstring s loc) }
+        if !handle_docstrings then
+          DOCSTRING (Docstrings.docstring s loc)
+        else
+          COMMENT (s, loc)
```

Comment:
This might be irrelevant, but shouldn't the s be replaced by ("*" ^ s)?  Or, if ocamldoc simply ignores the comment content, simply drop the token (in the loop inner function under token)?